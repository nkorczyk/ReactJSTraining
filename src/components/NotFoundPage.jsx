import React from 'react';
import styled from 'styled-components';

const NotFoundContainer = styled.div`
  margin: 160px;
  text-align: center;
`;

const NotFoundPage = () => (
  <NotFoundContainer>
    <h1>404 - Page Not Found</h1>
  </NotFoundContainer>
);

export default NotFoundPage;
