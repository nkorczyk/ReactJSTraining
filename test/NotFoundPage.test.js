import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../src/components/NotFoundPage';

describe('should render NotFoundPage component', () => {
  const element = shallow(<NotFoundPage />);
  it('Snapshot test with default props', () => {
    expect(element).toMatchSnapshot();
  });
});
