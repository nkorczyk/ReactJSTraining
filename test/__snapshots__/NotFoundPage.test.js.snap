// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage component Snapshot test with default props 1`] = `
<styled.div>
  <h1>
    404 - Page Not Found
  </h1>
</styled.div>
`;
